Bio/SeqIO/XdnaIO.py

```diff
@@ -36,6 +36,22 @@
 }
 
 
+def _read_header(handle, length):
+    """Read the specified number of bytes from the given handle.
+
+    Raise a ValueError("Empty file.") if the length of data read is zero. The
+    reason for having a separate function for the header is it enables raising
+    an empty file error if the length of the data read is zero. This might
+    not always be the case later in the file.
+    """
+    data = handle.read(length)
+    if len(data) == 0:
```

Comment:
Style wise,

```python
if not data:
```

i.e. PEP8 style, see earlier comment - where you could use the same helper. I wonder if this function is worth putting somewhere central like ``Bio/_utils.py``?

Reply:
Yeah, that would make sense. Have added it to Bio/_utils.py!